tests/conftest.py

```diff
@@ -150,6 +156,11 @@ def _bdb(_setup_database, _configure_bigchaindb):
     flush_db(conn, dbname)
 
 
+@pytest.fixture
+def _genesis(_bdb, genesis_block):
```

Comment:
Mmm, with `bdb` I think the config is implicit (since you need a config to set up an instance), but `genesis` might not imply `bdb` as `genesis` could just correlate with something else.

Another thought was to separate out the `genesis` marker from `bdb`, so that it was obvious you needed both (and would error if you only have `genesis`?), but this might be more work than worth.

Anyhow, not a big deal, I agree that after knowing what the marker does, it doesn't matter too much.